tests/utils/test_dataset_combine.py

```diff
@@ -173,32 +160,17 @@ def test_combine_datasets():
         minari.load_dataset(dataset_id) for dataset_id in test_datasets_ids
     ]
 
-    # testing without creating a copy
     combined_dataset = combine_datasets(
         test_datasets, new_dataset_id="cartpole-combined-test-v0"
     )
     assert combined_dataset.spec.env_spec.max_episode_steps is None
     _check_load_and_delete_dataset("cartpole-combined-test-v0")
 
-    # testing with copy
-    combined_dataset = combine_datasets(
-        test_datasets, new_dataset_id="cartpole-combined-test-v0", copy=True
-    )
-    assert combined_dataset.spec.env_spec.max_episode_steps is None
-    _check_load_and_delete_dataset("cartpole-combined-test-v0")
-
     # Check that we get max(max_episode_steps) when there is no max_episode_steps=None
     test_datasets.pop()
-    # testing without creating a copy
```

Comment:
What is the reason for removing these checks?

Reply:
It is not possible anymore to have copy=False, as it was causing problem (the ids of episode were not consistent)

Reply:
It is not specific about h5py isolation, but it is a bug in the current code: when you combine two datasets, the episode ids of the second dataset are modified when you have copy=False, i.e. they don't start anymore from 0